docs/dockerImages.md

```diff
@@ -23,14 +26,51 @@ Some questions may require additional customization of existing images, such as
    RUN pip install datascience
    ```
 
-5. In a terminal, change to the directory that contains the `Dockerfile` above and run the following commands (replacing `yourdockerhubaccount` with your Docker Hub account name, and `yourimagename` with an image name of your choice):
+5. In a terminal, change to the directory that contains the `Dockerfile` above and build the new image (replacing `yourdockerhubaccount` with your Docker Hub account name, and `yourimagename` with an image name of your choice):
 
    ```bash
    docker build --platform linux/amd64 -t yourdockerhubaccount/yourimagename .
+   ```
+
+   Note that the production environment used by PrairieLearn requires a `linux/amd64` platform for images. As such it is necessary that images are built using this platform, as listed in the `--platform` argument in the command above. This is particularly relevant for users of ARM-based architectures such as Apple Silicon or Windows on Arm systems, since these systems default to `linux/arm64` instead.
+
+6. Once the build process completes, push the image to the Docker Hub registry (again replacing `yourdockerhubaccount` and `yourimagename` as needed):
+
+   ```bash
    docker push yourdockerhubaccount/yourimagename
    ```
 
-6. Change the settings for the question that needs the custom image to use the image name you set above.
-7. In your course's "Sync" page, under "Docker images", find the image name above and click on "Sync".
+7. Change the settings for the question that needs the custom image to use the image name you set above.
+8. In your course's "Sync" page, under "Docker images", find the image name above and click on "Sync".
+
+Note that the process above _will not_ cause the image above to be automatically updated if there are changes or additional features to the original image. It is your responsibility to periodically update your custom image by repeating steps 5-8 above.
+
+### Testing custom images locally
+
+For simple custom images that only include one or two extra packages or libraries, the steps above should be sufficient to ensure that the image works as expected, testing them in production itself. Similarly, simple changes like updating package versions can be achieved by updating the files from step 4 above, then re-running steps 5-8.
```

Comment:
Nit, the other phrasing felt a bit clunky/unclear to me, specifically "testing them in production itself" at the end of the sentence. A potential change:

```suggestion
For simple custom images that only include one or two extra packages or libraries, following the steps above and testing the image in production may be sufficient to ensure that it works as expected. Similarly, simple changes like updating package versions can be achieved by updating the files from step 4 above, then re-running steps 5-8.
```

Feel free to iterate on this if you were trying to convey something different.